### ISSUE TITLE
Buchungsliste im Backende filtert falsch

### ISSUE THREAD

**Issue by hanneskasimir** [wielebenwir/commonsbooking#1057]
Das Enddatum filtert vermutlich auf 00:00 Uhr anstatt auf 23:59 Uhr.
![grafik](https://user-images.githubusercontent.com/28141760/170035896-988725d5-12cb-484f-8514-618c5ef43a1b.png)

**Comment by hansmorb** [comment]
Ich kann den Fehler nicht nachvollziehen. Die Liste filtert immer "AB" also wie hier in dem Beispiel alle Beginne ab dem 01.06., das Enddatum erkennt er aber auch richtig, beim 10.06. wird die Buchung unten nicht angezeigt, beim 11.06. schon:

![image](https://user-images.githubusercontent.com/6433480/172017037-d7decf89-63be-4370-ab0b-f7fd706ba6af.png)

**Comment by hanneskasimir** [comment]
sollte die Buchung beim 10.6. nicht auch angezeigt werden? der 10.6. gehört doch zum Zeitraum zwischen dem 1.6. und dem 10.6., oder?

**Comment by hansmorb** [comment]
Ne, das Enddatum dieser Buchung in diesem Beispiel ist eigentlich der 11.06, also stimmt die Berechnung hier schon.

**Comment by danielappelt** [comment]
Ich stimme @hanneskasimir zu. Bei uns im Team rechnet jeder damit, dass die Filterung "von ... bis ..." beide ausgewählten Tage mit einschließt und nicht nur das Startdatum. Die aktuelle Logik wird intuitiv bei uns von vielen nicht verstanden.

**Comment by chriwen** [comment]
@hanneskasimir und @danielappelt : Interessant. Wir hatten es so gesehen, dass der Filter den Zeitraum von - bis definiert. Alle Buchungen, die "komplett" innerhalb dieses Zeitraums liegen, also deren Startdatum gleich oder nach dem Start-Filter und deren Buchungsende vor oder am End-Datum des Filters liegt. 
Ihr interpretiert das so, dass alle Buchungen angezeigt werden, deren Startdatum innerhalb des Filter-Zeitraums liegt, richtig?

**Comment by hanneskasimir** [comment]
ja!

**Comment by danielappelt** [comment]
> Alle Buchungen, die "komplett" innerhalb dieses Zeitraums liegen, also deren Startdatum gleich oder nach dem Start-Filter und deren Buchungsende vor oder am End-Datum des Filters liegt.

Jein. Aktuell funktioniert es genauso nicht! Wenn ich als Enddatum den 11.6. auswähle, bekomme ich nur Buchungen angezeigt mit Enddatum bis einschließlich 10.6. Bei der Überprüfung wird anscheinend nicht <= 11.6. 23:59:59 verwendet sondern <= 11.6. 00:00:00.. zumindest vermute ich das.

Wenn es genau die Buchungen anzeigen würde, die mit den ausgewählten Start- und Enddatum übereinstimmen, würde das für uns vollkommen ausreichen. Aktuell ist es aber, meine ich, tatsächlich schon so, dass Start- und Enddatum innerhalb der ausgewählten Daten liegen müssen, damit eine Buchung ausgegeben wird.

**Comment by hansmorb** [comment]
Dann sehe ich das mal als feature an, da es eine neue Art der Filterung ist (Filterung in einem Datumszeitraum)
